typings/index.d.ts

```diff
@@ -10,12 +10,12 @@ import { getQueriesForElement, BoundFunction } from './get-queries-for-element';
 declare const within: typeof getQueriesForElement;
 
 interface Query extends Function {
-  (testRenderer: ReactTestRenderer | NativeTestInstance, ...args: any[]):
+  (testRenderer: ReactTestRenderer, ...args: any[]):
```

Comment:
I think this may be backwards. A query only takes a NativeTestInstance, never a ReactTestRenderer.

Reply:
Oh I just thought testRenderer should be `ReactTestRenderer` by seeing these changes.

https://github.com/manakuro/native-testing-library/commit/d8c3b30da218163349e2d22e6be63e4e5214b2e0#diff-0da64c26f363af99c40bbae0dc3fb0bfR8

Should it be replaced with `NativeTestInstance` in `typings/queries.d.ts` like this ?

```.ts
export type QueryByBoundProp = (
  testRenderer: NativeTestInstance,
  id: Matcher,
  options?: MatcherOptions,
) => NativeTestInstance | null;
...
```

and Query just takes `NativeTestInstance` ?

```.ts
interface Query extends Function {
  (testRenderer: NativeTestInstance, ...args: any[]):
    | Error
    | Promise<NativeTestInstance[]>
    | Promise<NativeTestInstance>
    | NativeTestInstance[]
    | NativeTestInstance
    | null;
}
```

Reply:
Yep this is right. And to make it more clear, we might want to change the name of the argument from `testRenderer` as a part of this as well 👍